flags.go

```diff
@@ -18,12 +18,12 @@ func flags(config *Config) flag.FlagSet {
 type stringMap map[string]string
 
 func (v stringMap) Set(val string) error {
-	spl := strings.SplitN(val, ":", 2)
-	if len(spl) != 2 {
+	spl := strings.Split(val, ":")
+	if len(spl) <= 1 {
 		return errors.New("import flag must be of form path:alias")
 	}
 
-	v[spl[0]] = spl[1]
+	v[strings.Join(spl[:len(spl)-1], ":")] = spl[len(spl)-1]
```

Comment:
would it work to use strings.LastIndex here to avoid the join? something like:

~~~~golang
i := strings.LastIndex(val, ":")
if i < 1 {
  return errors.New("import flag must be of form path:alias")
}
v[val[:i]] = val[i+1:]
~~~~

Reply:
good idea